PhpAmqpLib/Wire/IO/SocketIO.php

```diff
@@ -87,15 +74,15 @@ public function connect()
     }
 
     /**
-     * @return resource
+     * @inheritdoc
      */
-    public function getSocket()
+    public function get_socket()
```

Comment:
This change isn't backwards-compatible. Can we keep the old name for now?

Reply:
We can, just for sake of compatibility :)

Reply:
Thanks! Otherwise this PR could only go into version `3.0.0`